### ISSUE TITLE
Allow calling javascript functions with a JSON parameter from an SQL query

### ISSUE THREAD

**Issue by mrft** [orientechnologies/orientdb#2334]
For example, when calling the function 

```
function jsonparameterTest( par )
  if ( typeof par == 'object' ) {
    return JSON.stringify( par );
  }
  else { 
    return "NO OBJECT: " + par;
  }
}
```

in the studio like this:

```
select jsonParameterTest( 'mystringparameter' ) FETCHPLAN *:1
```

It works as expected.

But I would like to do something like this:

```
select jsonParameterTest( { "a": 1, "b": "zE" } ) FETCHPLAN *:1
```

**Comment by luigidellaquila** [comment]
this is already supported by the query parser, but not by the underlying executor, so I have to postpone it to next release

**Comment by luigidellaquila** [comment]
Implemented in v 3.0. 

Closing, thanks

Luigi
